From aac153ed68db0b6391d201fcea70bde593cb5f63 Mon Sep 17 00:00:00 2001
From: Konsta <konsta09@gmail.com>
Date: Sat, 28 Sep 2013 13:20:52 +0300
Subject: [PATCH] add atlas40 to config.sh

---
 config.sh |    5 +++++
 1 file changed, 5 insertions(+)

diff --git a/config.sh b/config.sh
index e60cf5c..d27c9bf 100755
--- a/config.sh
+++ b/config.sh
@@ -101,6 +101,11 @@ case "$1" in
 	repo_sync $1
 	;;
 
+"atlas40")
+	echo DEVICE=$1 >> .tmp-config &&
+	repo_sync inari
+	;;
+
 "otoro"|"unagi"|"keon"|"inari"|"leo"|"hamachi"|"peak"|"helix"|"wasabi")
 	echo DEVICE=$1 >> .tmp-config &&
 	repo_sync $1
-- 
1.7.9.5

